src/lib/idmap/sss_idmap.h

```diff
@@ -1049,10 +1052,13 @@ enum idmap_error_code sss_idmap_offset_murmurhash3(void *pvt,
 
 /**
  * Structure for private data for offset_murmurhash3. If not given 0xdeadbeef
- * will be used as seed.
+ * will be used as seed. UTF8 strings will be normalized by default but not
+ * casefolded.
  */
 struct sss_idmap_offset_murmurhash3_data {
     uint32_t seed;
+    bool normalize;
+    bool casefold;
```

Comment:
Do I understand correctly that `casefold` includes `normalize` and thus combination ((casefold = true) & (normalize = false)) doesn't make a sense?

Reply:
Yes, I added this to the comment.